tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -3645,6 +3645,84 @@ def _prepare_inputs(
             num_accepted_tokens_device, req_id_to_old_request, resource_manager,
             maybe_graph)
 
+    def _prepare_encoder_inputs(self,
+                               inputs: Dict[str, Any]) -> Dict[str, Any]:
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)